app/routes/users/components/StudentConfirmation.js

```diff
@@ -101,7 +105,67 @@ const StudentConfirmation = ({
               inputValue="komtek"
             />
           </RadioButtonGroup>
+          <RadioButtonGroup
+            name="isTwoYears"
+            label={
+              <Flex>
+                <div>Begynner du på to-årig master?</div>
```

Comment:
Could this be question be "Begynner du på 2-åring eller 5-åring master?" with alternatives "2-åring" and "5-årig" (with 5-årig as default checked) instead? I think it's a bit clearer/ less chance of people getting confused when the question is not biased towards one or the other